### PR TITLE
Docs: Propose fix typo for function

### PR DESCRIPTION
**Pull request by jeis2497052**
**What is the purpose of this pull request? (put an "X" next to item)**

[ X] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

**What changes did you make? (Give an overview)**

**Is there anything you'd like reviewers to focus on?**

### CODE REVIEW

**Comment by abiduzz420** [comment]
Hello @jeis2497052 
You need to follow the commit-message guidelines. The check fails because you need to start the commit message with label `Docs:` . Also edit the PR title. Example: `Docs: Propose fix typo for function`

**Comment by jeis2497052** [comment]
Sorry, can I repair this PR or start again?

**Comment by jeis2497052** [comment]
Not sure if or how to repair?

**Comment by abiduzz420** [comment]
PR title could be directly edited from the UI. For making changes to the commit message, you could take the help of this: https://help.github.com/articles/changing-a-commit-message/

**Comment by jeis2497052** [comment]
is the UI edit OK now with Docs: added?

**Comment by platinumazure** [comment]
Hi @jeis2497052, thanks for contributing!

Our commit message check will look at either the first commit's commit message, or the PR title, depending on the number of commits in the PR. If the PR has 1 commit, we look at the commit message; if 2 or more, we look at the PR title. This is because GitHub's squash and merge interface uses the exact same logic to decide what commit message to suggest when we are about to merge.

Right now, your PR title matches our guidelines, but the commit message does not, and there is only one commit.

I think we could handle this using one of the following approaches:

1. Ignore the issue, and we'll make sure the commit message matches the PR title on merge
1. You could find another thing to change (ideally in the same file) and push another commit, which will then make our commit message check look at the PR title
1. You could use `git commit --amend`, then `git push (remote-name) (branch-name) -f`, as suggested by @abiduzz420.

The last option is probably the "best" option, but also the most annoying on your end. Let me know how you would like to proceed.

**Comment by jeis2497052** [comment]
Did I properly amend ?

**Comment by platinumazure** [comment]
@jeis2497052 Yes! Everything looks great now and our commit-message status check is happy :smile: Thanks so much for taking the time to do this, it really helps us out and it makes our changelog beautiful.

TravisCI and AppVeyor need to rerun due to the forced push, but hopefully those will pass soon enough.

**Comment by ilyavolodin** [comment]
Thanks for the pull request.

File: docs/rules/valid-typeof.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Examples of **incorrect** code for this rule:
 typeof foo === "strnig"
 typeof foo == "undefimed"
 typeof bar != "nunber"
-typeof bar !== "fucntion"
+typeof bar !== "function"
```
Comment (Wilfred) [inline docs/rules/valid-typeof.md]: @platinumazure @jeis2497052 I think this is wrong. These are deliberate typos, aren't they?

Comment (not-an-aardvark) [reply docs/rules/valid-typeof.md]: Agreed, would you like to create a PR to fix this example?

Comment (Wilfred) [reply docs/rules/valid-typeof.md]: #10459 :)

**Pull request merged**